src/configure/helper/repoAnalysisHelper.ts

```diff
@@ -1,24 +1,28 @@
 import * as path from 'path';
+import { ModaClient } from '../clients/modaClient';
 import { PortalExtensionClient } from "../clients/portalExtensionClient";
 import { AzureSession, GitRepositoryParameters, RepositoryAnalysisApplicationSettings, RepositoryAnalysisParameters, RepositoryAnalysisRequest, RepositoryDetails, RepositoryProvider, SupportedLanguage } from "../model/models";
 import { RepoAnalysisConstants } from "../resources/constants";
+import { RedirectLinkHelper, ServiceFramework } from './redirectLinkHelper';
+
 
 export class RepoAnalysisHelper {
     private portalExtensionClient: PortalExtensionClient;
+    private modaClient: ModaClient;
```

Comment:
maintaining two different clients is problematic. We should define an interface for the repositoryAnalysisClient and then have two implementations of that - PortalExtensionRespositoryAnalysisClient and ModaRepositoryAnalysisClient. Once such a client is created, the interface will help in using it anywhere in code, otherwise everywhere we have to pass both portalExtensionClient and modaClient